src/main/app/embedContent/EmbedContent.ts

```diff
@@ -20,9 +20,10 @@ export class EmbedContent {
     }
 
     testFixUrl(tests) {
+        var url = this.url;
         var result = _.find(tests, function (test) {
```

Comment:
Change this to `test =>` and it will fix this as well. We should be using arrow functions here, but I must have missed this one.